uportal-war/src/main/java/org/apereo/portal/shell/PortalShell.java

```diff
@@ -76,9 +76,6 @@ public static void main(String[] args) throws Exception {
                         new CompilerConfiguration(System.getProperties());
                 final GroovyShell shell = new GroovyShell(binding, conf);
                 shell.run(scriptFile, args);
-            } else {
```

Comment:
@ChristianMurphy -- The zero-arg method wasn't on `groovy.lang.GroovyShell`;  it was on `org.codehaus.groovy.tools.shell.Groovysh`.  The API docs for that class are not in the same collection as your link (for some reason);  they are here:  http://docs.groovy-lang.org/docs/groovy-2.4.10/html/gapi/

Oddly, I can't find a zero-arg version of the `Groovysh.run()` method in either the 2.4.10 version _or_ the 2.2.2 version of Groovy.

I can, however, find the Ant behavior that connects to the line you removed:  https://github.com/Jasig/uPortal/blob/master/build.xml#L1078.  If we remove the line, I should think we remove the stuff in Ant that calls it.

But I wish we didn't have to remove it.  At first look, it seems like the `run(String evalString, List<String> filenames)` overload will work for us, since the docs say "Run Interactive Shell with **optional** initial script and files to load" (emphasis mine).

Lastly -- if the line does go in the end, it was the only reference to that class.  We should also remove the `import org.codehaus.groovy.tools.shell.Groovysh;` at the top.